api4/file.go

```diff
@@ -285,6 +285,7 @@ func getFilePreview(c *Context, w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		c.Err = err
 		c.Err.StatusCode = http.StatusNotFound
+		return
```

Comment:
It prevents to call the `fileReader.Close()` when is nil, because the open failed.